Extension/src/packageManager.ts

```diff
@@ -356,7 +350,7 @@ export class PackageManager {
         });
     }
 
-    private InstallPackage(pkg: IPackage): Promise<void> {
+    private async InstallPackage(pkg: IPackage): Promise<void> {
         this.AppendLineChannel(localize("installing.package", "Installing package '{0}'", pkg.description));
 
         return new Promise<void>((resolve, reject) => {
```

Comment:
This async function is creating a Promise.  It's contents could  be moved into the enclosing function instead, right?

Reply:
let's discuss about this, will call you